docs/03_For_Hosts/07_Customizations.md

```diff
@@ -1,9 +1,9 @@
-RSS-Bridge ships a few options the host may or may not activate. All options are currently defined in the [index.php](https://github.com/RSS-Bridge/rss-bridge/blob/master/index.php) file. This means they'll be reset after upgrading RSS-Bridge!
```

Comment:
This section should probably be merged with `08_Custom_Configuration.md` now that all configuration is customizable using a config file but I did not really feel like it so I just updated the obviously outdated parts.